AcademyResidentInformationApi/V1/Infrastructure/Person.cs

```diff
@@ -12,38 +12,39 @@ public class Person
     {
         [Column("claim_id")]
         [Key]
-        public int ClaimId { get; set; }
+        public int? ClaimId { get; set; }
```

Comment:
Could having `ClaimId` and `HouseRef` nullable have an effect on the composite key?

Reply:
I was also unsure about this but the tests run fine and they are nullable in the actual database and I think we should be mirroring that here.